src/Hazelcast.Net/Networking/NetworkAddress.cs

```diff
@@ -302,6 +305,32 @@ internal static bool TryParse(string s, out NetworkAddress address, int defaultP
         /// <returns>A new instance of the network address with the same host but a different port.</returns>
         internal NetworkAddress WithPort(int port) => new NetworkAddress(Host, port);
 
+        /// <summary>
+        /// Determines whether this address is reachable, by trying to connect to it.
+        /// </summary>
+        /// <param name="timeout">A timeout.</param>
+        /// <returns><c>true</c> if the connection was successful; otherwise false.</returns>
+        /// <remarks>Use a timeout value of -1ms for infinite.</remarks>
```

Comment:
.NET has a constant you can reference in such cases: `System.Threading.Timeout.InfiniteTimeSpan`

Reply:
yes but I prefer to be explicit here